Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM debian
+
+LABEL key="georgios@paradigm.xyz"
```

Comment:
Is adding this label useful? Its a non standard label name for a maintainer, perhaps the label name should be maintainer like what the alpine image does, see https://hub.docker.com/r/alpine/git/dockerfile/